python-template/{{cookiecutter.placeholder_repo_name}}/.gitignore

```diff
@@ -0,0 +1,78 @@
+*.py[cod]
+__pycache__
+.pytest_cache
+
+# C extensions
+*.so
+
+# Packages
+*.egg
+*.egg-info
+dist
+build
+eggs
+parts
+bin
+var
+sdist
+develop-eggs
+.installed.cfg
+lib
+lib64
+
+# Installer logs
+pip-log.txt
+
+# Unit test / coverage reports
+.cache/
+.pytest_cache/
+.coverage
+.coverage.*
+.tox
+coverage.xml
+htmlcov/
+
+
+
+# Translations
+*.mo
+
+# IDEs and text editors
+*~
+*.swp
+.idea/
+.project
+.pycharm_helpers/
+.pydevproject
+
+# The Silver Searcher
+.agignore
+
+# OS X artifacts
+*.DS_Store
+
+# Logging
+log/
+logs/
+chromedriver.log
+ghostdriver.log
+
+# Complexity
+output/*.html
+output/*/index.html
+
+# Sphinx
+docs/_build
+docs/modules.rst
+docs/{{ cookiecutter.sub_dir_name }}.rst
+docs/{{ cookiecutter.sub_dir_name }}.*.rst
+
+# Private requirements
+requirements/private.in
+requirements/private.txt
+
+# tox environment temporary artifacts
+tests/__init__.py
+
+# Development task artifacts
+default.db
```

Comment:
I don't know what this is, but it seems awfully vague for something that will apply to all of our repos by default.

Reply:
I think its django database related, gonna remove from this